lib/jenkins_pipeline/job.rb

```diff
@@ -31,7 +31,7 @@ def serialize
     private
 
     def build_status
-      return "unknown" if %w(running unknown failure).include?(@upstream_status)
+      return "unknown" if %w(running unknown failure aborted).include?(@upstream_status)
```

Comment:
maybe we can invert this logic? because the only one missing in this array is the success. So, compare if it's not success would be less lines more clear.